packages/generator-common/src/options-parser.ts

```diff
@@ -110,6 +110,45 @@ export function resolveRequiredPath<GeneratorOptionsT>(
     : resolve(arg.toString());
 }
 
+/**
+ * Resolves input string using glob notation.
+ * @internal
+ * @param arg - Path argument as passed by the user.
+ * @param options - Options as passed by the user.
+ * @returns Absolute path as a `string`.
+ */
+export function resolveInputGlob<GeneratorOptionsT>(
+  arg: string,
+  options: GeneratorOptionsT & { config?: string }
+): string[] {
+  const inputPath = options.config
+    ? resolve(dirname(options.config), arg.toString())
+    : resolve(arg.toString());
+  return getInputFilePaths(inputPath.split(sep).join(posix.sep));
```

Comment:
[pp] When I workd on the [glob](https://github.com/SAP/cloud-sdk-js/blob/2650859415f7bc54de39caa7bf9156700dc8a628/packages/generator-common/src/options-parser.ts#L87) resolving I used the `cwd` property of the glob. To merge before has the same effect - We should align the two ways. Either remove the `cwd` in the mentioned method or use it also here to have the code as consistent as possible.

Reply:
Removed the `cwd`. Also fixed the tests that were affected as a result